src/main/java/in/projecteka/devservice/DevServiceConfiguration.java

```diff
@@ -109,6 +116,14 @@ public Credential credentialDisabled() {
         return new GoogleCredential();
     }
 
+    @SneakyThrows
+    @Bean({"supportRequestCredential"})
+    public Credential supportRequestCredential(SupportRequestProperties supportRequestProperties) {
+        List<String> SCOPES = Collections.singletonList(SheetsScopes.SPREADSHEETS);
+        return GoogleCredential.fromStream(new FileInputStream(supportRequestProperties.getCredentialPath()))
+                .createScoped(SCOPES);
+    }
+
```

Comment:
We can use the existing googlecredentials instead of creating a new.. because it uses the same credentials.json (Service Account)

Reply:
Sure, Done